### DISCUSSION TITLE
esp32s2-evse-d-a with single phase

### DISCUSSION THREAD

**Discussion by nagyrobi** [dzurikmiroslav/esp32-evse#54] [General]
The `esp32s2-evse-d-a` board is intended for usage with 3 phase - but would it be possible to use it (temporarily) with only 1 phase? I mean, only with software modifications and external circuitry but no new PCB/schematic/board mods?

**Comment by dzurikmiroslav** [comment]
I thing yes, board has 4 digital output (12V). One can control relay witch will be driving HV Contactor on L2 & L3.

**Reply by nagyrobi** [reply]
It's simply the case that the site where the unit will be installed currently only has 1 phase, but will be upgraded to 3 phase next year. So  the goal would be to assemble the unit for 3 phase, but only use with 1 phase configuration until then.

**Reply by dzurikmiroslav** [reply]
I thought you meant dynamic switching between 1/3 phases. 
Of course, it can also be used for one phase.

**Reply by nagyrobi** [reply]
Actually dynamic switching is also not a bad idea...

**Reply by nagyrobi** [reply]
> Of course, it can also be used for one phase.

Any special consideration, setting? Won't the voltage and current measurements (Energy meter Type) cause errors because only one phase is connected at the input of the board?
